app/assets/javascripts/lib/router.coffee

```diff
@@ -1,9 +1,24 @@
 class @Router extends Backbone.Router
   routes:
+    "demand/:sidebar(/:slide)" : "demand"
```

Comment:
@marliekeverweij , I don't really get the changes in these lines (3-8). Are you sure they need to be there?

Reply:
You're right! They should not be there. I've deleted them now.